src/frontend/components/CallAssignment.js

```diff
@@ -132,20 +133,32 @@ export class CallAssignment extends React.Component {
   }
 
   renderIntervieweeInfo() {
-    let interviewee = this.props.currentUser.intervieweeForCallAssignment;
-    let name = this.intervieweeName();
-    let number = this.formatPhoneNumber(interviewee.phone)
+    let interviewee = this.props.currentUser.intervieweeForCallAssignment
+    let name = this.intervieweeName()
+    let formattedNumber = this.formatPhoneNumber(interviewee.phone)
+    let plivoStatusText = this.state.plivoStatusText;
+    let plivoCallInProgress = this.state.plivoCallInProgress;
 
     let sideBar = (
-      <div style={{
-        ...BernieText.default,
-        color: BernieColors.blue,
-        fontSize: '1.5em',
-        fontWeight: 600
-      }}>
-        {name}
-        <br />
-        {number}
+      <div>
+        <div style={{
+          ...BernieText.default,
+          color: BernieColors.blue,
+          fontSize: '1.5em',
+          fontWeight: 600
+        }}>
+          {name}
+          <br />
+          <a href={`tel:+1${interviewee.phone}`}
+             style={{color: BernieColors.blue,
+                     textDecoration: 'none'}}>
+            {formattedNumber}
+          </a>
+          <PlivoDialer number={`1${interviewee.phone}`}
+            endpointUsername='bernie151217012337'
```

Comment:
@saikat No domain verification as far as I can tell. Their own examples include usernames and passwords used client-side, either user-entered or hardcoded. Those are per-app credentials that can easily be discarded, is I guess their thinking?